src/OrchardCore.Modules/OrchardCore.Media.Indexing/Startup.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Indexing;
+using OrchardCore.Modules;
+
+namespace OrchardCore.Media.Indexing;
+
+public class Startup : StartupBase
+{
+    public override void ConfigureServices(IServiceCollection services)
+    {
+        services.AddScoped<IContentFieldIndexHandler, MediaFieldIndexHandler>();
+    }
+}
+
+[Feature("OrchardCore.Media.Indexing.Pdf")]
+public class PdfStartup : StartupBase
```

Comment:
Why do we need to have `OrchardCore.Media.Indexing` while we creating  a separate modules for media indexing?

Reply:
The whole folder can be removed.

Reply:
yes. I removed it from the project but forgot to remove the folder. I'll clean that up and rename the module as @Skrypt suggested.